README.md

```diff
@@ -7,21 +7,25 @@
 > **Note:** This project is still in its early stages and is not yet ready for production use. It is currently being developed as a just-for-fun project. If you are interested in contributing, please feel free to open an issue or a pull request.
 
 ## Installation
+
 - You can install the latest release from [crates.io][crates.io] using `cargo install cmus-notify`
 - You can also install the latest version from the git repository using `cargo install --git https://github.com/anas-elgarhy/cmus-notify`
-- Or if you use Arch Linux, you can install the [AUR package][aur package] using your favorite AUR helper. e.g. `yay -S notify-cmus`
+- Or if you use Arch Linux, you can install the [AUR package][aur package] using your favorite AUR helper. e.g. `yay -S cmus-notify`
```

Comment:
The Arch user repo already has a package with the name `cmus-notify` and I discovered this after I choose the name for this project unfortunately.
So I have to use a different  name or reverse the name order for the AUR package

Reply:
oh sorry about that, I clicked the link and it didn't pull up a package, so had a look at the link and I saw of a name was reversed and thought it was an oversight. I will revert that back.

Maybe cmus-notify-rs? Although it may cause confusion because people they think it's a rest version of AntoineGagne/cmus-notify, that's a tough one.